### ISSUE TITLE
trouble running example code

### ISSUE THREAD

**Issue by leoohyama** [simonpcouch/anyflights#14]
Hi, I have tried to run the example code from this package's main github page, using both the CRAN and developer version of 'anyflights' and I keep running into this issue where the code:
```
library(anyflights)
pdxflights19<-get_flights("PDX", 2019,6)
```

Results in this:
> pdxflights19 <- anyflights("PDX", 2019, 6)
                                Total Time Elapsed          
Finished Processing Arguments                   1s                         0s
Downloaded Flights Data for June               35s          
Finished Downloading Flights Data              36s          
  Downloading Airlines...                         Error in open.connection(3L, "rb") : HTTP error 500.
In addition: Warning message:
One or more parsing issues, see `problems()` for details 

It seems like the flight data downloads fine but the airline data doesn't. I also ran the "get_flights" then the "get_airlines" function which resulted in this :

`pdxflights19 <- anyflights("PDX", 2019)
`
> pdxflights19<-get_flights("PDX", 2019,6)
                                Total Time Elapsed          
Finished Processing Arguments                   6s                         0s
Downloaded Flights Data for June               44s          
Finished Processing Flights Data               46s          
All Done!                                         Warning messages:
1: In for (i in seq_along(specs)) { :
  closing unused connection 4 (http://www.transtats.bts.gov/Download_Lookup.asp?Lookup=L_UNIQUE_CARRIERS)
2: In mget(objectNames, envir = ns, inherits = TRUE) :
  restarting interrupted promise evaluation
3: One or more parsing issues, see `problems()` for details 

`pdxairlines19<-get_airlines(flights_data =pdxflights19)`
> pdxairlines19<-get_airlines(flights_data =pdxflights19)
Error in open.connection(4L, "rb") : HTTP error 500.

Not sure if this a problem on the user end or the server end? I have tried doing this on other wifi connection as well.

**Comment by simonpcouch** [comment]
Thanks for the issue! I'm seeing the same 500 error with `get_airlines()`, though not with `get_flights()`.

500 usually means something server-side is out of wack. I'll give this a week to make sure this isn't a temporary issue, and if not, will put some time in on a fix. :-)

**Comment by joshua-kerr** [comment]
I was wondering if there was a fix for this that you discovered.  I am still having the same issue.  Any luck?

**Comment by simonpcouch** [comment]
Looks like this hasn't resolved itself :-( I'll try to make time for this sooner than later, though would gladly review a PR if others are able to troubleshoot this issue in the meantime!

**Comment by simonpcouch** [comment]
Long time coming (apologies), but addressed in #15! Thanks all.
